generalized_linear_regression/generalized_linear_regression.md

```diff
@@ -0,0 +1,533 @@
+<!--
+
+module_id: generalized_linear_regression
+author:   Rose Hartman
+email:    hartmanr1@chop.edu
+version:  1.0.0
+current_version_description: Initial version
+module_type: standard
+docs_version: 4.0.0
+language: en
+narrator: UK English Female
+mode: Textbook
+
+title: Generalized Linear Regression
+
+comment:  What is generalized linear regression (including logistic regression) and when might you need it? 
+
+long_description: This lesson walks you through what generalized linear models are, and how they are similar to and different from ordinary linear regression. It uses logistic regression as a specific example, but touches on other generalized linear models as well. This is a theoretical overview, not hands-on coding practice, so no experience with any particular software is expected. If you want to understand generalized linear models better, regardless of what statistical software you use, this is for you!
+
+estimated_time_in_minutes: 60
+
+@pre_reqs
+Learners should already be familiar with the following concepts in statistics and math:
+
+- linear regression (also called "ordinary least squares (OLS) linear regression")
+- the equation of a line (intercept and slope)
+@end
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
+- describe how generalized linear models are similar to and different from linear models
+- define a "link function"
+- identify when a logistic regression model is appropriate
+@end
```

Comment:
This should probably also include a LO about identifying boundedness and continuity-ness  of variables! you've already got the quiz questions to cover that content, so I don't think it would be any change other than adding here.

Reply:
Elizabeth and I beefed up a lot of the boundedness/continuity material with the idea that it was supporting LO 3, identify when logistic regression is appropriate (and, to a lesser extent, LO 1). 
We could add another LO for it, but I think my inclination is not to, as I doubt that those things per se will feel valuable to learners (I don't think anyone looking at our modules will be like how can I learn about "boundedness and continuity?" It's just that they actually really need to understand both of those things to know when they need to move to a generalized linear model). Since the LOs are for the overview, I think they're more useful when they speak to skills and topics a potential learner might already be able to parse, without having done the module first.

Reply:
Hmm, I see where you're coming from! 
The hang up I have (and reason I brought this up) is the QA rule stating "All major topics covered are represented by a learning objective -- there's no significant knowledge imparted that isn't specified in the learning objectives" -- continuity and boundedness seem like significant bits of knowledge, especially given the extent to which we cover them. 

Do you think we might, in the future, write other modules in  in which knowing how to determine boundedness and continuity might be a pre-req? If so, I'd lean toward thinking that having it as a LO might be helpful? 

That being said, I know some of the QA things are clear cut (like the presence of alt text), and others (like this one) are a bit more subjective, so I could be convinced that this shouldn't be a 4th LO if even after re-reading that QA line item you're still not convinced it makes sense to add :)

Reply:
Re whether continuity/boundedness could come up as a future pre-req, that's a great question... I'm think I'm leaning towards "no", at least not in the way they're treated here -- this discussion is really tightly calibrated to how features of your outcome variable impact whether or not you can use a linear model. For other applications (like when to run an ANOVA, or whether to do a Welch approximation for your t-test, etc.), we'd want a more general discussion of different kinds of variables (and for that I generally link to Kahn Academy lessons anyway -- I don't think we need to write that content). I don't think the way it's explained here would really be particularly useful outside of potential future modules on linear models or generalized linear models, and in that case the relevant pre-req would be something much closer to the LOs that are already articulated here, especially LO 1.
Re the QA requirement: If I were to add an LO for continuity and boundedness, I think I'd suggest something like "identify whether or not a variable can be treated as continuous and unbounded", but for me that is basically the first half of "identify when a logistic regression model is appropriate" (in my head the GLM decision chart is something like "is your outcome continuous and unbounded? yes, use OLS. no, then is it binary? yes, then use logistic regression, no then look at other GLMs." :) )
That said, if you still feel like the module would benefit from LO 4, then let's put it in! :)

Reply:
Hmm, good points! 

I think I'm convinced we don't need to add it!